server_addon/nuke/server/settings/publish_plugins.py

```diff
@@ -165,7 +165,7 @@ class BakingStreamModel(BaseSettingsModel):
         title="Custom tags", default_factory=list)
 
 
-class ExtractReviewDataMovModel(BaseSettingsModel):
```

Comment:
This change will also require an increase of patch version in `server_addon\nuke\server\version.py`

Reply:
We will also need to preserve the original plugin settings in place, but will need to add some annotation for users to understand the future depreciation. For more inspiration about this look to `server_addon\nuke\server\settings\workfile_builder.py:61` Here it is added as docstrings and since those are shown in the settings it could be good way to inform user about the changes.